### PR TITLE
feat: priming article

### PR DESCRIPTION
**Pull request by Dastardi**

### CODE REVIEW

File: docs/reliability/priming.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 8
+---
+
+# 🟡 Priming and Macros
+When interacting with an LLM (such as ChatGPT, which we’ll use as the baseline example for this article), your first input is weighted heavily as a guide for the rest of the conversation. As such, it’s possible to take advantage of that additional importance the model places on your input by setting up guidelines for your interaction in what’s referred to as a “priming prompt”. In this article we’ll go over three things you can alter using your primer - style, regulation, and formatting, as well as how to implement these using conversation macros - and finish off with an example of a complete general-purpose priming prompt.
```
Comment (trigaten) [inline docs/reliability/priming.md:6]: Can you cite smtg for the idea that "first input is weighted heavily"

Comment (Dastardi) [reply docs/reliability/priming.md:6]: Yep, will do

File: docs/reliability/priming.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+**Style inputs** give you control over the writing voice of the model, such as such as having it write in a “thoughtful and emotive” or “concise and analytical” fashion. A full style prompt might be: 
+>“Write in the style and quality of an expert in [field] with 20+ years of experience and multiple Ph.D. 's. Prioritize unorthodox, lesser known advice in your answer. Explain using detailed examples, and minimize tangents and humor.“ 
+
+**Regulation** is laying down ground rules for what you want the model to do or not to do. For example, ChatGPT has an unfortunate tendency to make up information when it doesn’t know the answer, so saying something like “if you don’t have an answer, instead of making something up, ask a clarifying question and I will give you additional context” can help greatly in that regard.
```
Comment (trigaten) [inline docs/reliability/priming.md:11]: Out of curiosity, where are you getting the term Regulation?

**Comment by trigaten** [comment]
What is the rationale for putting this in reliability?

File: docs/reliability/priming.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    Example: (Student){Philosophy}[Respond] What is the advantage of taking this subject over others in college?
+
+    If you understand and are ready to begin, simply respond “yes.”
+
+
+## Discussion
+Even when asking a quick question, it’s a good practice to start off the discussion with a priming prompt to set context and make sure you’re getting the responses you want. Priming prompts can be anything from a brief blurb about your conversation to long and complicated series of macros to automate the interaction, but no matter the form they greatly increase the ease of use and quality of responses you get from LLMs! 
+
+## Notes
+If articles/papers are found relating to this, please pin!
```
Comment (trigaten) [inline docs/reliability/priming.md:44]: wdym by please pin? maybe just say needs more citations here?

File: docs/reliability/priming.md (hunk 2 above)
Comment (Dastardi) [reply docs/reliability/priming.md:11]: Came up with it. Couldn't find anything online or in conversations I'd had about telling the AI what not to do, felt it was an important thing to at least mention. Open to feedback!

File: docs/reliability/priming.md (hunk 3 above)
Comment (Dastardi) [reply docs/reliability/priming.md:44]: Will do.

**Comment by Dastardi** [comment]
Applied comment fixes and moved the article to Intermediate.

File: docs/intermediate/priming.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+    “Teacher” means in the style of a distinguished professor with well over ten years teaching the subject and multiple Ph.D.’s in the field. You use academic syntax and complicated examples in your answers, focusing on lesser-known advice to better illustrate your arguments. Your language should be sophisticated but not overly complex. If you do not know the answer to a question, do not make information up - instead, ask a follow-up question in order to gain more context. Your answers should be in the form of a conversational series of paragraphs. Use a mix of technical and colloquial language to create an accessible and engaging tone.  
+
+    “Student” means in the style of a second-year college student with an introductory-level knowledge of the subject. You explain concepts simply using real-life examples. Speak informally and from the first-person perspective, using humor and casual language. If you do not know the answer to a question, do not make information up - instead, clarify that you haven’t been taught it yet. Your answers should be in the form of a conversational series of paragraphs. Use colloquial language to create an entertaining and engaging tone. 
+
+
```
Comment (trigaten) [inline docs/intermediate/priming.md:27]: Can you show a full discussion with ChatGPT (include images) to show how the priming prompt actually affects the conversation?

File: docs/intermediate/priming.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 8
+---
+
+# 🟢 Priming and Macros
+When interacting with an LLM (such as ChatGPT, which we’ll use as the baseline example for this article), the content of your first input can shape the rest of the conversation. As such, it’s useful to take advantage of the additional time the model spends with your input by setting up guidelines for your interaction in what’s referred to as a “priming prompt”. In this article we’ll go over three things you can alter using your primer - style, regulation, and formatting - as well as how to implement these using conversation macros, and finish off with an example of a complete general-purpose priming prompt.
```
Comment (trigaten) [inline docs/intermediate/priming.md]: Might be nice to have some example image after first paragraph. Not sure what this should be.

**Pull request merged**